build.sbt

```diff
@@ -3,15 +3,17 @@ description := "An example of GraphQL server written with Play and Sangria."
 
 version := "1.0-SNAPSHOT"
 
-scalaVersion := "2.12.6"
+scalaVersion := "2.13.3"
+
+val sangriaVersion = "2.0.1"
 
 libraryDependencies ++= Seq(
   guice,
   filters,
-  "org.sangria-graphql" %% "sangria" % "1.4.2",
-  "org.sangria-graphql" %% "sangria-slowlog" % "0.1.8",
-  "org.sangria-graphql" %% "sangria-play-json" % "1.0.4",
-  "org.scalatest" %% "scalatest" % "3.0.5" % "test")
+  "org.sangria-graphql" %% "sangria" % sangriaVersion,
+  "org.sangria-graphql" %% "sangria-slowlog" % sangriaVersion,
+  "org.sangria-graphql" %% "sangria-play-json" % sangriaVersion,
```

Comment:
Those libraries can have different release cycle.
Maybe we should not have one value for all of them?